orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestrator.java

```diff
@@ -208,7 +208,7 @@ public void bootstrap(List<GatewayConfig> allGatewayConfigs, Set<Node> targets,
             Set<String> allTargets = targets.stream().map(Node::getPrivateIp).collect(Collectors.toSet());
             uploadSignKey(sc, primaryGateway, gatewayTargets, allTargets, exitModel);
             OrchestratorBootstrap saltBootstrap = saltBootstrapFactory.of(sc, saltConnectors, allGatewayConfigs, targets, params);
-            Callable<Boolean> saltBootstrapRunner = saltRunner.runner(saltBootstrap, exitCriteria, exitModel);
+            Callable<Boolean> saltBootstrapRunner = saltRunner.runnerWithConfiguredErrorCount(saltBootstrap, exitCriteria, exitModel);
```

Comment:
@lacikaaa Please confirm that it wont cause any issues if we change this during bootstrap and rebootstrap.

Reply:
what is the current change? what are the changed values? I can't see anything related in the commit message

Reply:
Hi @lacikaaa , the commit message has this "change bootstrap maxError to be configured." Basically this is what we discussed in this jira: https://jira.cloudera.com/browse/CB-20809 We'd like to reduce the number of retries when salt bootstrap fails/has errors. Let us know! Thank you so much!

Reply:
so please include the reason in the commit message, include what is the current config, what is the new config. if it's in the jira already, then great, you only have to copy it. thanks

Reply:
Hi @lacikaaa Included more details in the commit. Please let me know if it looks good to you. Thank you!

Reply:
thanks, 20 seems reasonable to me